internal/common/context.bzl

```diff
@@ -0,0 +1,328 @@
+load("//internal/common/actions:actions.bzl", "create_source_dir", "run_js")
+load("//internal/npm_packages:rule.bzl", "NpmPackagesInfo")
+# load("@rules_nodejs//internal/common:node_module_info.bzl", "NodeModuleInfo", "collect_node_modules_aspect")
```

Comment:
What was this needed for?

Reply:
I will take it out, I was attempting to get this to integrate with the `rules_nodejs` [fine grained package dependencies](https://github.com/bazelbuild/rules_nodejs#fine-grained-npm-package-dependencies).  But based on https://github.com/bazelbuild/rules_nodejs/issues/57 they will probably be going with another path and don't actually make the `collect_node_modules_aspect` public.